src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/resource/NonAssociationsResource.kt

```diff
@@ -63,16 +64,22 @@ class NonAssociationsResource(
     @Schema(description = "The offender prisoner number", example = "A1234BC", required = true)
     @PathVariable
     prisonerNumber: String,
-  ): NonAssociationDetails {
-    return NonAssociationDetails(prisonerNumber)
+  ): PrisonerNonAssociations {
+    return nonAssociationsService.getPrisonerNonAssociations(
+      prisonerNumber,
+      NonAssociationOptions(
+        onlySamePrison = true,
+        includeClosed = false,
+      ),
+    )
   }
 
   @PostMapping("/non-associations")
   @PreAuthorize("hasRole('ROLE_NON_ASSOCIATIONS') and hasAuthority('SCOPE_write')")
   @ResponseStatus(HttpStatus.CREATED)
   @Operation(
     summary = "Creates a non-association between two prisoners.",
-    description = "Requires ROLE_NON_ASSOCIATIONS role with write scope.",
+    description = "Requires ROLE_NON_ASSOCIATIONS role with write scope. Also requires ROLE_GLOBAL_SEARCH or ROLE_PRISONER_SEARCH role.",
```

Comment:
The API should have its own creds to give it access to these APIs - the user calling this application would not

Reply:
https://github.com/ministryofjustice/hmpps-non-associations-api/pull/35/commits/2d6f8b6a6752a1008daad07a4d47555c5df1f2ff